scripts/config.ts

```diff
@@ -332,6 +332,7 @@ function writeL2ChainConfig(argv: any) {
 		"eip150Block": 0,
 		"eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
 		"eip155Block": 0,
+                "espresso": argv.espresso,
```

Comment:
I suddenly thought of a question: if we modify the chain config here, does that mean we can't use integration for the already running chain? This seems inappropriate.

Reply:
Ah ok. Anyway it seems not a big deal here since we can move this flag to validators' or sequencers' config at any time.